lib/matplotlib/markers.py

```diff
@@ -274,9 +274,7 @@ def set_fillstyle(self, fillstyle):
         """
         if fillstyle is None:
             fillstyle = rcParams['markers.fillstyle']
-        if fillstyle not in self.fillstyles:
-            raise ValueError("Unrecognized fillstyle %s"
-                             % ' '.join(self.fillstyles))
+        cbook._check_in_list(self.fillstyles, fillestyle=self.fillstyles)
```

Comment:
I don't understand how this is supposed to work if you don't pass `fillstyle`. Should it be:
```suggestion
        cbook._check_in_list(self.fillstyles, fillstyle=fillstyle)
```

Reply:
woops, sorry, fixed.